config/config.go

```diff
@@ -20,6 +20,9 @@ var FilterDatasetControllerURL = "http://localhost:20001"
 // The URL of the Geography controller
 var GeographyControllerURL = "http://localhost:23700"
 
+// Geography feature is enabled
+var GeoEnabled = false
```

Comment:
pedantic, but possibly worth making this `GeographyEnabled` so its consistent with `GeographyControllerURL`